src/autohost.ts

```diff
@@ -325,6 +326,43 @@ export class Autohost implements TachyonAutohost {
 			);
 		}
 	}
+
+	shutdown(): void {
+		// Force shutdown (i.e. pressing CTRL+C, CTRL+C in sequence), kill all games then exit.
+		if (this.shuttingDown) {
+			this.logger.warn(
+				'second shutdown signal recieved, forcing shutdown by killing all games and exiting',
+			);
+			this.gamesMgr.killAllGames();
+			process.exit(0);
```

Comment:
When I was thinking about it, I thought that we would not call `process.exit` at random point, but integrate shutdown into the loop in main, and let that terminate cleanly. This give additional check of sorts, that all async operations finished and we succesfully closed everything down.